package.json

```diff
@@ -50,7 +50,6 @@
     "graphql-iso-date": "^3.6.1",
     "jest-watch-typeahead": "^0.3.1",
     "lint-staged": "^7.3.0",
-    "nexus": "file:.",
```

Comment:
What was this even achieving, I forget now.

Reply:
Making the import of `import { core } from "nexus"` work here:

https://github.com/prisma-labs/nexus/pull/227/files#diff-2d9f20d621ccd8f7a36d3d1bb829f8e8